h/indexer/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from h.indexer.reindexer import reindex
```

Comment:
It doesn't look like any refers to "h.indexer.reindex", so this (and the `__all__`) probably aren't needed.